hydropandas/observation.py

```diff
@@ -813,6 +813,64 @@ def from_artdino_file(cls, path=None, **kwargs):
 
         return cls(measurements, meta=meta, **meta)
 
+    @classmethod
+    def from_waterconnect(
+        cls,
+        dh_no,
+        meta_series=None,
+        tmin=None,
+        tmax=None,
+        only_metadata=False,
+        verify=True,
+        pumping=True,
+        anomalous=True,
+    ):
+        """Read data from waterinfo csv-file or zip.
```

Comment:
When is the data loaded from CSV and when from a zipfile?

Reply:
Not really this was an old docstring. For now only API reading is supported